application/view/omeka/admin/item/sidebar-select.phtml

```diff
@@ -64,13 +64,18 @@ $expanded = $resourceClassId || $itemSetId || $id;
     </div>
 
     <?php if ($itemsFound): ?>
-    <button type="button" class="quick-select-toggle"><?php echo $translate('Quick add'); ?></button>
+    <span class="sr-only quick-select-on success" tabindex="-1"><?php echo $translate('Quick add checkboxes are now available.'); ?></span>
+    <span class="sr-only quick-select-off success" tabindex="-1"><?php echo $translate('Quick add checkboxes are now hidden.'); ?></span>
+    <button type="button" class="quick-select-toggle"><?php echo $translate('Quick add'); ?> <span class="sr-only active"><?php echo $translate('Is active'); ?></span><span class="sr-only inactive"><?php echo $translate('Inactive'); ?></span></button>
+    <span class="sr-only select-all-on success" tabindex="-1"><?php echo $translate(sprintf('%s All results are now selected.', $translate('Quick add checkboxes are now available.'))); ?></span>
```

Comment:
Reversed order of the translate and sprintf calls here (the first or "outer" one specifically).